src/components/projects/view/application-hour-tile.svelte

```diff
@@ -9,12 +9,19 @@
   export let activity;
   export let application;
   export let handlePanelOpen;
+  $: report = application.reports.find(report => report.reporter_email === account.email);
 
   const dispatch = createEventDispatcher();
 </script>
 
 <AccordionSection on:panel-open={handlePanelOpen} let:toggle>
-  <button slot="handle" class="btn handle round" type="button" on:click={toggle}>
+  <button
```

Comment:
Why is it lowercase? Does that still make it our imported component or the native HTML element?

Reply:
Native. The difference is that this one doesn't have a ripple attached, which was messing things up

Reply:
Or, actually, no. The reason why the native element is used here is because of the slot. But there's one more place in code where a native button is preferred for ripple reasons